docs/jotforms-integration.md

```diff
@@ -168,8 +148,6 @@ From this iframe src URL.
 `230793530776059` is the ID for the form, and should always be the first path segment of the iframes src URL.
 
 
-
-
 <!-- TODO: remove legacy gudiance once SB guidance is complete in NXT-839 -->
```

Comment:
Think we're fine to remove all this as this PR being merged means the legacy stuff isn't required